assets/js/translator.js

```diff
@@ -460,6 +460,11 @@ function populateTranslationList() {
         numDstCols = Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS) < (TRANSLATION_LIST_COLUMNS + 1)
             ? Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS)
             : TRANSLATION_LIST_COLUMNS;
+
+    var thresholdWindowWidth = 1000;
+    numSrcCols = $(window).width() < thresholdWindowWidth && numSrcCols === TRANSLATION_LIST_COLUMNS ? numSrcCols - 1 : numSrcCols;
```

Comment:
> This is a little bit worrying since the ideal is a dynamically adjusting number, not one that simply decrease by one sometimes.

As I discussed in my approach, um, this will suffice as we will only face issues wherein the number of columns is 4 ( anything less than that will not hurt ). Thus, dynamically adjusting this number to 1 less based on the said conditions ( and also incrementing it otherwise ), should perhaps solve this!